external/header.php

```diff
@@ -161,8 +178,8 @@ function () {
             $requestTimeFloat[1] = 0;
         }
 
-        $requestTs = array('sec' => $time, 'usec' => 0);
-        $requestTsMicro = array('sec' => $requestTimeFloat[0], 'usec' => $requestTimeFloat[1]);
+        $requestTs      = array('sec' => $time,                 'usec' => 0);
```

Comment:
I'm not sure changes like this are useful ones. Keeping these arrays space aligned will add noise/work to future changes.

Reply:
The idea was to pick non-functional changes out of there, to make diff smaller in that pull request.

But I guess it's better just squash merge changes in that branch to single commit. Making clean commit history just not justified and takes too much energy.